tests/test_unit/test_queue/test_delete_queue.py

```diff
@@ -57,10 +55,11 @@ async def test_superuser_can_delete_queue(
     }
     assert result.status_code == 200
 
-    queue = await session.get(Queue, group_queue.id)
-    await session.refresh(queue)
-
-    assert queue.is_deleted
+    # Assert queue was deleted
+    stmt = select(Queue).where(Queue.id == q_id)
+    res = await session.execute(stmt)
+    queue_in_db = res.scalars().first()
+    assert queue_in_db is None
```

Comment:
```suggestion
    queue_in_db = await session.get(Queue, q_id)
    assert queue_in_db is None
```

Reply:
this doesn't work as  test session doesn't see updates commited by backend session, therefore `queue_in_db` will  still be an existing record even though it is deleted in db and we cannot neither `session.expire_all()` or `session.commit()` as this session is used continiosly in fixtures to delete created records

Reply:
What about https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.expunge?